recipes/libcudss/meta.yaml

```diff
@@ -0,0 +1,157 @@
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcudss", max_pin="x.x.x") }}
+    files:
+      - lib/libcudss.so  # [linux]
+      - lib/libcudss_commlayer_nccl.so  # [linux]
+      - lib/libcudss_commlayer_openmpi.so  # [linux]
+      - lib/cudss.lib  # [win]
+      - include/cudss*
+      - lib/cmake/cudss
+    requirements:
+      host:
+        - {{ pin_subpackage("libcudss", exact=True) }}
+      run:
+        - {{ pin_subpackage("libcudss", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -f $PREFIX/include/cudss.h  # [linux]
+        - test -f $PREFIX/include/cudss_distributed_interface.h  #[linux]
+        - test -L $PREFIX/lib/libcudss_commlayer_nccl.so  # [linux]
+        - test -L $PREFIX/lib/libcudss_commlayer_openmpi.so  # [linux]
+        - test -L $PREFIX/lib/libcudss.so  # [linux]
+        - test ! -f $PREFIX/lib/libcudss_static.a  # [linux]
+        - if not exist %LIBRARY_LIB%\\cudss.lib exit 1  # [win]
+        - if not exist %LIBRARY_INC%\\cudss.h exit 1  # [win]
+        - if not exist %LIBRARY_INC%\\cudss_distributed_interface.h exit 1  # [win]
+    about:
+      home: https://developer.nvidia.com/cudss
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/cudss/license.html
+      summary: >-
+        NVIDIA cuDSS is an optimized, first-generation
+        GPU-accelerated Direct Sparse Solver library for solving linear systems
+        with very sparse matrices.
+      description: >-
+        NVIDIA cuDSS is an optimized, first-generation
+        GPU-accelerated Direct Sparse Solver library for solving linear systems
+        with very sparse matrices. Direct Sparse Solvers are an important part
+        of numerical computing for real-time applications like autonomous
+        driving and process simulation, where increasing complexity and high
+        throughput demands a robust direct solver.
+      doc_url: https://docs.nvidia.com/cuda/cudss/
+
+about:
+  home: https://developer.nvidia.com/cudss
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
+  license_file: LICENSE
+  license_url: https://docs.nvidia.com/cuda/cudss/license.html
+  summary: >-
+    NVIDIA cuDSS is an optimized, first-generation
+    GPU-accelerated Direct Sparse Solver library for solving linear systems
+    with very sparse matrices.
+  description: >-
+    NVIDIA cuDSS is an optimized, first-generation
+    GPU-accelerated Direct Sparse Solver library for solving linear systems
+    with very sparse matrices. Direct Sparse Solvers are an important part
+    of numerical computing for real-time applications like autonomous
+    driving and process simulation, where increasing complexity and high
+    throughput demands a robust direct solver.
+  doc_url: https://docs.nvidia.com/cuda/cudss/
+
+extra:
+  recipe-maintainers:
+    - conda-forge/cuda
```

Comment:
I also wonder if it makes sense to get one of the cudss developers (me) into the maintainers list.

Reply:
```suggestion
    - conda-forge/cuda
    - kvoronin
```